pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/SmId.java

```diff
@@ -49,6 +44,12 @@ public SmId(Integer smIdPk, String smIdType, String smIdValue, Integer tissueId)
         this.tissueId = tissueId;
     }
 
+    public SmId(Integer smIdPk, String smIdValue, Integer tissueId) {
```

Comment:
`smIdPk` should be an int--looking at these constructor usages, they are all called with `getInt()`, which returns an `int`, so it should be safe to changed `smIdPk` in the constructors to `int` and then change calling classes to use `int foo = rs.getInt(...)`.

Reply:
Correct! Method is auto-generated by intellij and I overlooked the types, will fix